base/rational.jl

```diff
@@ -156,6 +156,10 @@ widen(::Type{Rational{T}}) where {T} = Rational{widen(T)}
 Approximate floating point number `x` as a [`Rational`](@ref) number with components
 of the given integer type. The result will differ from `x` by no more than `tol`.
 
+!!! note
+    `rationalize` does inexact conversion; it produces a `Rational` value equal to the
```

Comment:
Did you mean
```suggestion
    `rationalize` does inexact conversion; it produces a `Rational` value which is not equal to the
```
?
The first sentence says it is an approximation. Perhaps enough to add something like
```
For exact conversion use the `Rational` constructor.
```

Reply:
I'd say: **which is equal decimal value x**, because the decimal value for example `1.2` is not necessarily the "real" floating-point value as its represented in memory. `1.2` in memory might be:
```julia
julia> big(1.2)
1.1999999999999999555910790149937383830547332763671875
```

So when I said decimal value, I was specifying that's it just equal to the written value and not its internal representation. Moreover, this is how @StefanKarpinski puts it in the article I highlighted above in this PR.

Is it confusing? If it is, then it's best to just drop it and then go for the "For exact conversion use the `Rational` constructor." you suggested.

Reply:
I'd leave out mention of decimal values because folks may arrive at floating point numbers very close to rational numbers with small denominators through a variety of means, many of which don't involve writing out decimal numbers. For example `rationalize(1/7) == rationalize(0.14285714285714285) == 1//7` while `1//7 ≠ 0.14285714285714285`.

Reply:
So maybe using "floating point" numbers would be better?

Reply:
DONE!!!